### DISCUSSION TITLE
Revanced Duolingo still getting ads

### DISCUSSION THREAD

**Discussion by Cryptnym** [ReVanced/revanced-discussions#1067] [Question]
Both my father and i are using the Duolingo app but are still seeing ads, how do you switch them off? I've looked in the settings and have switched all the relevant options off but are still seeing ads.

**Comment by oSumAtrIX** [comment]
No Duolingo ads patch is released yet. Official links at https://revanced.app

**Comment by LisoUseInAIKyrios** [comment]
@Cryptnym you and your family are using a counterfeit bootleg ReVanced Manager.  Uninstall everything you installed and change your account passwords.

**Reply by Cryptnym** [reply]
Thanks for the advice, done as instructed
